views/lexicon/forms/button-group.html.twig

```diff
@@ -216,24 +230,25 @@
             'guidance': 'Help text to give more information about expected input',
             'items': [
                 {
-                    'id': 'foo9',
-                    'label': 'Foo',
-                    'name': 'foo-9',
+                    'id': 'foo-' ~ i,
+                    'label': 'Foo ' ~ i,
+                    'name': 'foo-' ~ i,
                     'checked': true
                 },
                 {
-                    'id': 'bar9',
-                    'label': 'Bar',
-                    'name': 'foo-9'
+                    'id': 'bar-' ~ i,
+                    'label': 'Bar ' ~ i,
+                    'name': 'foo-' ~ i
                 },
                 {
-                    'id': 'baz9',
-                    'label': 'Baz',
-                    'name': 'foo-9'
+                    'id': 'baz-' ~ i,
+                    'label': 'Baz ' ~ i,
+                    'name': 'foo-' ~  i
                 }
             ]
         })
     }}
+    {% set i = i + 1 %}
```

Comment:
If we're going to the trouble of making this half programmatic would it be worth going the whole hog sticking these in a loop?